src/pkg/bundle/create.go

```diff
@@ -26,6 +28,32 @@ func (b *Bundle) Create() error {
 		return err
 	}
 
+	// Populate values from valuesFiles if provided
```

Comment:
I was about to make a comment asking why this is in `create` vs `deploy` but I think I understand. The nuance is that this values file is only for chart defaults